src/lib/viewers/controls/media/MediaSettingsControls/MediaSettingsRate.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
```

Comment:
Should this file be `MediaSettingsMenuAutoplay`?

Reply:
Whoops, this component is meant to come in with the next PR. Sorry about that.